src/admin/test_pagerduty.rs

```diff
@@ -1,6 +1,6 @@
 use anyhow::Result;
 use clap::Clap;
-use failure::_core::str::FromStr;
+use std::str::FromStr;
```

Comment:
huh, looks like IntelliJ autocompleted some interesting imports when I did the refactoring 🙈 

guess these two changes would make sense independent from the `oauth2` upgrade :)